pkg/acme/cache.go

```diff
@@ -123,10 +125,3 @@ func buildHTTPClient(skipTLSVerify bool) *http.Client {
 		Timeout: time.Second * 30,
 	})
 }
-
-var timeout = 5 * time.Second
-
-func dialTimeout(ctx context.Context, network, addr string) (net.Conn, error) {
-	d := net.Dialer{Timeout: timeout}
-	return d.DialContext(ctx, network, addr)
```

Comment:
Previously we'd construct a new `net.Dialer` each time `DialContext` was called on the Transport - I'm not sure if it makes much difference, but this _could_ help? 🤷‍♂ not too sure, but this is definitely more correct anyway 😄